site/src/sphinx/server-basics.rst

```diff
@@ -143,6 +143,36 @@ You might be interested in decorating a service using other decorators, for exam
 
 You can also use an arbitrary object that's annotated by the ``@Path`` annotation using ``annotatedService()``.
 
+Per service configuration
+-------------------------
+
+The examples above are just mapping the path of an HTTP request on a service. If you want to set configuration
+for a specific service, you can use fluent API:
+
+.. code-block:: java
+
+    ServerBuilder sb = new ServerBuilder();
+    sb.route()                                   // Configure the service.
+      .post("/foo/bar")                          // Matched when the path is "/foo/bar" and the method is POST.
+      .consumes(JSON)                            // Matched when the "content-type" header is "application/json".
```

Comment:
How about adding `import` or specifying `MediaType.JSON` from here?

Reply:
Will use `MediaType.JSON` and add the import statement because we still need it. Thanks! :-)

Reply:
Changed. Didn't add the import statement because we already did that in the previous example. :-)